src/Mentat.UI/Controllers/CardController.cs

```diff
@@ -48,27 +52,27 @@ public ActionResult Create()
         public ActionResult Create(IFormCollection collection)
         {
             try
-            {
-                Models.MongoHelper.ConnectToMongoService();
-                Models.MongoHelper.FlashCards =
-                               Models.MongoHelper.database.GetCollection<Models.Card>("FlashCards");
-                
+            {                
                 object id = GenRandomId(24);
-
-                Models.MongoHelper.FlashCards.InsertOneAsync(new Models.Card
+                cardService.Create(new Models.Card
                 {
                   _id = id,
-                  notes = collection["notes"]
-                });  
+                  notes = collection["notes"],
```

Comment:
Here is one location that will need to be updated once the properties are renamed to PascalCase.

Reply:
there may be other locations as well, but this is the last one I will explicitly notate